src/utils/createArtifactsContent.js

```diff
@@ -114,6 +114,8 @@ export const createModelsRowData = (artifact, project, showExpandButton) => {
       {
         id: `key.${artifact.ui.identifierUnique}`,
         header: 'Name',
+        headerId: 'name',
+        isSortable: true,
```

Comment:
for now we don't need artifact headers to be sortable. So, you can delete `isSortable` from this function